lib/sisimai/bite/email/courier.rb

```diff
@@ -0,0 +1,265 @@
+                # Action: failed
+                v['action'] = cv[1].downcase
+
+              elsif cv = e.match(/\A[Ss]tatus:[ ]*(\d[.]\d+[.]\d+)/)
+                # Status: 5.1.1
+                # Status:5.2.0
+                # Status: 5.1.0 (permanent failure)
+                v['status'] = cv[1]
+
+              elsif cv = e.match(/\A[Rr]emote-MTA:[ ]*(?:DNS|dns);[ ]*(.+)\z/)
+                # Remote-MTA: DNS; mx.example.jp
+                v['rhost'] = cv[1].downcase
+                if v['rhost'] =~ / /
+                  # Get the first element
+                  v['rhost'] = v['rhost'].split(' ').shift
+                end
+
+              elsif cv = e.match(/\A[Ll]ast-[Aa]ttempt-[Dd]ate:[ ]*(.+)\z/)
+                # Last-Attempt-Date: Fri, 14 Feb 2014 12:30:08 -0500
+                v['date'] = cv[1]
+
+              else
+                if cv = e.match(/\A[Dd]iagnostic-[Cc]ode:[ ]*(.+?);[ ]*(.+)\z/)
+                  # Diagnostic-Code: SMTP; 550 5.1.1 <userunknown@example.jp>... User Unknown
+                  v['spec'] = cv[1].upcase
+                  v['diagnosis'] = cv[2]
+
+                elsif p =~ /\A[Dd]iagnostic-[Cc]ode:[ ]*/ && cv = e.match(/\A[ \t]+(.+)\z/)
+                  # Continued line of the value of Diagnostic-Code header
+                  v['diagnosis'] ||= ''
+                  v['diagnosis']  += ' ' + cv[1]
+                  havepassed[-1] = 'Diagnostic-Code: ' + e
+                end
+              end
+
+            else
+              # This is a delivery status notification from marutamachi.example.org,
+              # running the Courier mail server, version 0.65.2.
+              #
+              # The original message was received on Sat, 11 Dec 2010 12:19:57 +0900
+              # from [127.0.0.1] (c10920.example.com [192.0.2.20])
+              #
+              # ---------------------------------------------------------------------------
+              #
+              #                           UNDELIVERABLE MAIL
+              #
+              # Your message to the following recipients cannot be delivered:
+              #
+              # <kijitora@example.co.jp>:
+              #    mx.example.co.jp [74.207.247.95]:
+              # >>> RCPT TO:<kijitora@example.co.jp>
+              # <<< 550 5.1.1 <kijitora@example.co.jp>... User Unknown
+              #
+              # ---------------------------------------------------------------------------
+              if cv = e.match(/\A[>]{3}[ ]+([A-Z]{4})[ ]?/)
+                # >>> DATA
+                next if commandtxt.size > 0
+                commandtxt = cv[1]
+
+              elsif cv = e.match(/\A[Rr]eporting-MTA:[ ]*(?:DNS|dns);[ ]*(.+)\z/)
+                # Reporting-MTA: dns; mx.example.jp
+                next if connheader['rhost'].size > 0
+                connheader['rhost'] = cv[1].downcase
+                connvalues += 1
+
+              elsif cv = e.match(/\A[Rr]eceived-[Ff]rom-MTA:[ ]*(?:DNS|dns);[ ]*(.+)\z/)
```

Comment:
__[rubocop]__
* Style/IfInsideElse: Convert `if` nested inside `else` to `elsif`.

<!-- Comment by [SideCI](https://sideci.com) -->